search/intermediate_result.cpp

```diff
@@ -76,33 +108,35 @@ void ProcessMetadata(FeatureType const & ft, Result::Metadata & meta)
   meta.m_isInitialized = true;
 }
 
-PreResult1::PreResult1(FeatureID const & fID, PreRankingInfo const & info) : m_id(fID), m_info(info)
+PreRankerResult::PreRankerResult(FeatureID const & fID, PreRankingInfo const & info)
+  : m_id(fID), m_info(info)
 {
   ASSERT(m_id.IsValid(), ());
 }
 
 // static
-bool PreResult1::LessRank(PreResult1 const & r1, PreResult1 const & r2)
+bool PreRankerResult::LessRank(PreRankerResult const & r1, PreRankerResult const & r2)
 {
   if (r1.m_info.m_rank != r2.m_info.m_rank)
     return r1.m_info.m_rank > r2.m_info.m_rank;
   return r1.m_info.m_distanceToPivot < r2.m_info.m_distanceToPivot;
 }
 
 // static
-bool PreResult1::LessDistance(PreResult1 const & r1, PreResult1 const & r2)
+bool PreRankerResult::LessDistance(PreRankerResult const & r1, PreRankerResult const & r2)
 {
   if (r1.m_info.m_distanceToPivot != r2.m_info.m_distanceToPivot)
     return r1.m_info.m_distanceToPivot < r2.m_info.m_distanceToPivot;
   return r1.m_info.m_rank > r2.m_info.m_rank;
 }
 
-PreResult2::PreResult2(FeatureType const & f, m2::PointD const & center, m2::PointD const & pivot,
-                       string const & displayName, string const & fileName)
+RankerResult::RankerResult(FeatureType const & f, m2::PointD const & center,
```

Comment:
Could you please separate definitions by "// RankerResult ----------- ...``` ?

Reply:
done